src/applications/appeals/995/content/notice5103.jsx

```diff
@@ -34,6 +34,7 @@ export const Notice5103Description = ({ onReviewPage }) => {
         onCloseEvent={hideAlert}
         showIcon
         visible={visibleAlert}
+        uswds
```

Comment:
Is this `<VaAlert>` different than `<va-alert>`. I see in the docs that `<va-alert>` is the only one with `uswds` prop.

Reply:
Because we're using `onCloseEvent`, we need the React bindings, so that's why it's using `<VaAlert>`